lib/facts/ubuntu/path.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module Facter
+  module Ubuntu
+    class Path
+      FACT_NAME = 'path'
+
+      def call_the_resolver
+        fact_value = Facter::Resolvers::Path.resolve(FACT_NAME)
```

Comment:
It would be great if you could use a symbol here, in some cases the name we sent to the resolver is different from the fact name.
```suggestion
        fact_value = Facter::Resolvers::Path.resolve(:path)
```

Reply:
@IrimieBogdan changed here: 8b0acd3 
Thanks :)